src/main/java/com/ifmo/epampractice/serviceimpl/AdminOrdersServiceImpl.java

```diff
@@ -0,0 +1,143 @@
+package com.ifmo.epampractice.serviceimpl;
+
+import com.ifmo.epampractice.dao.CarDamageDao;
+import com.ifmo.epampractice.dao.InvoiceDao;
+import com.ifmo.epampractice.dao.OrderDao;
+import com.ifmo.epampractice.dao.UsersDao;
+import com.ifmo.epampractice.daoimpl.CarDamageDaoImpl;
+import com.ifmo.epampractice.daoimpl.InvoiceDaoImpl;
+import com.ifmo.epampractice.daoimpl.OrderDaoImpl;
+import com.ifmo.epampractice.daoimpl.UsersDaoImpl;
+import com.ifmo.epampractice.entity.CarDamageEntity;
+import com.ifmo.epampractice.entity.InvoiceEntity;
+import com.ifmo.epampractice.entity.OrderEntity;
+import com.ifmo.epampractice.entity.UsersEntity;
+import com.ifmo.epampractice.enums.DamageStatus;
+import com.ifmo.epampractice.enums.InvoiceStatus;
+import com.ifmo.epampractice.enums.OrderStatus;
+import com.ifmo.epampractice.services.AdminOrdersService;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.List;
+
+public class AdminOrdersServiceImpl implements AdminOrdersService {
+
+    @Override
+    public List<OrderEntity> getAllOrders() {
+        OrderDao dao = new OrderDaoImpl();
+        return dao.getAll();
+    }
+
+    @Override
+    public OrderEntity getOrderById(int orderId) {
+        OrderDao orderDao = new OrderDaoImpl();
+        return orderDao.get(orderId);
+    }
+
+    @Override
+    public void approveOrder(int orderId) {
+        OrderDao orderDao = new OrderDaoImpl();
+        OrderEntity orderEntity = orderDao.get(orderId);
+        orderEntity.setStatus(OrderStatus.APPROVED);
+        orderDao.update(orderEntity);
+    }
+
+    @Override
+    public void disapproveOrder(int orderId) {
+        OrderDao orderDao = new OrderDaoImpl();
+        OrderEntity orderEntity = orderDao.get(orderId);
+        orderEntity.setStatus(OrderStatus.DENIED);
+        orderDao.update(orderEntity);
+    }
+
+    @Override
+    public boolean userDocumentsIsAvailable(int userId) {
+        UsersDao usersDao = new UsersDaoImpl();
+        UsersEntity usersEntity = usersDao.get(userId);
+        return usersEntity.getDrivingLicenseId() != 0 && usersEntity.getPassId() != 0;
```

Comment:
Are you sure `0` and `null` is the same for this field? What happens if someone creates DrivingLicense with id=0?